### DISCUSSION TITLE
Welcome to OpenSarToolkit Discussions!

### DISCUSSION THREAD

**Discussion by 12rambau** [ESA-PhiLab/OpenSarToolkit#72] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by alexisahedo** [comment]
Hello everyone,

Its nice to meet you all, mi name is Alexis Ahedo, a Geological Engineer working in Mexico, I’m starting to work with SAR satellite images in a new project, aiming to produce information at national scale. I’m new to python programming so it has been very helpful to me your code in the develop of the fist stages of the project. I hope to keep working in the feature with you all.

Alexis A

**Reply by BuddyVolly** [reply]
Dear Alexis, 

great to hear. Most of us are actually working now for FAO, specifically the SEPAL platform. If you work at national scale and you are not very familiar with coding, this might be of interest for you as well. There is some no-coding environment for Google Earth Engine, also for the creation of S1 mosaics. 
register for free and login at: sepal.io
documentation: docs.sepal.io
google group: https://groups.google.com/g/sepal-users

And a video in Spanish of how easy it is to create S1 mosaics. 
https://www.youtube.com/watch?v=a0j47L8f3a4

Hasta pronto,
BV

**Reply by alexisahedo** [reply]
Dear BuddyVolly,

Thanks a lot for the warm welcome, it wasn’t on my radar the SEPAL project, I’ll check it out with great expectations.
I’m also working with an Open Data Cube infrastructure, learning a lot. 
I have seen the trend of in the Cloud processing, hopefully in the near future I’ll also join.

Saludos amigo.

Alexis A.
